app/assets/javascripts/app.js

```diff
@@ -1,10 +1,14 @@
 import './libs';
 import ProjectsIndexController from './controllers/projects/IndexController';
+import TagGroupsController from './controllers/tag_groups/controller';
```

Comment:
This file should have a better name like `TagGroupsController`, not just `Controller`.